src/resource/resource.ts

```diff
@@ -176,6 +180,29 @@ export function isPodOwner(
   return podOwner === webId;
 }
 
+/**
+ *  Given an solidDataset, fetch the urls of all contained resources.
+ *  If the solidDataset given is not a container, or is missing resourceInfo, throw an error.
+ *
+ * @param solidDataset The container from which to fetch all contained resource urls
+ * @returns A list of urls, each of which points to a contained resource of the given solidDataset
+ * @since 1.2.0
```

Comment:
We'll update this version number when we make a new release, because the new version number depends on what else goes into that release.

```suggestion
 * @since Not released yet.
```